### PR TITLE
Eslint

### PR DESCRIPTION
**Pull request by nthitz**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -48,3 +23,34 @@ function updateUi() {
...
+  })
+
+  // populate
+  ui.setGeos(mapData.getUniqueFeatureIds())
+  ui.setDatasetLabels(data.getLabels())
+  selectDataset(data.getDataset(0))
+  updateUi()
+}
+
+
```
Comment (adamflorin) [inline index.js]: Heh, no lint rule about two newlines in a row, huh :trollface:

Comment (nthitz) [reply index.js]: It's configurable! defaults to a max of 2 http://eslint.org/docs/rules/no-multiple-empty-lines but we could make it one!

Comment (adamflorin) [reply index.js]: Oh neat! I prefer 1 personally. What do you think?

File: source/components/DatasetSelector.js [hunk 2]
```diff
@@ -81,6 +86,7 @@ export default class DatasetSelector extends React.Component {
 DatasetSelector.propTypes = {
   labels: React.PropTypes.array,
   onDatasetSelected: React.PropTypes.func,
+  onCustomDataset: React.PropTypes.func,
```
Comment (adamflorin) [inline source/components/DatasetSelector.js:88]: We may as well add `onCustomDataset` to `defaultProps`, too. I don't suppose there's a way to lint for that as well?

Comment (nthitz) [reply source/components/DatasetSelector.js:88]: https://github.com/yannickcr/eslint-plugin-react/issues/528 doesn't seem like it yet :(

**Pull request merged**